src/testutils/blockchain_tests.rs

```diff
@@ -486,16 +486,24 @@ macro_rules! bdk_blockchain_tests {
                 let (wallet, blockchain, descriptors, mut test_client) = init_single_sig();
 
                 wallet.sync(&blockchain, SyncOptions::default()).unwrap();
-                assert_eq!(wallet.get_balance().unwrap(), 0);
+                assert_eq!(wallet.get_balance().unwrap().untrusted_pending, 0);
```

Comment:
nit: this should be `get_total()`? It doesn't seem to me that there's a specific reason for using untrusted_pending, and we expect to have all the balance to zero at this point

Reply:
That was like so because (1) when that line was added, `get_total()` didn't exist, and (2) Since I was checking untrusted pending later (equal to 50,000) I thought I should show only it to be equal to 0 before. I should probably change it to `get_total()`, because of... stricter testing 😅